stubs/python-jose/jose/jws.pyi

```diff
@@ -4,21 +4,21 @@ from typing import Any
 from .backends.base import Key
 
 def sign(
-    payload: str | Mapping[str, Any],
+    payload: bytes | Mapping[str, Any],
```

Comment:
It is passed down to `base64url_encode`, which expects `bytes`.
- https://github.com/mpdavis/python-jose/blob/96474ecfb6ad3ce16f41b0814ab5126d58725e2a/jose/jws.py#L42
- https://github.com/mpdavis/python-jose/blob/96474ecfb6ad3ce16f41b0814ab5126d58725e2a/jose/jws.py#L153